File: Problem_1/problem1.pl
%! [problem1].
%! problem1(X)

problem1(Sum) :- findall(N,num(N),L), sumlist(L,Sum).

num(N) :- between(1,999,N), multiple(N).

multiple(N) :- (N rem 3) =:= 0, !.
multiple(N) :- (N rem 5) =:= 0.

File: 01-09/Problem_1/problem1.c
#include <stdio.h>

// gcc problem1.c -o problem1
int main() {
    int sum = 0;
    for (int i = 3; i < 1000; i++) {
        if (i % 3 == 0 || i % 5 == 0) {
            sum += i;
        }
    }
    printf("%d\n", sum);
    return 0;
}